cluster-scope/base/rbac.authorization.k8s.io/clusterroles/sre-read-cr/clusterrole.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sre-group-read
+rules:
+- apiGroups: [""]
```

Comment:
can you convert these to yaml so it's consistent with the other clusterroles in this path